pyvista/plotting/mapper.py

```diff
@@ -763,28 +770,28 @@ class FixedPointVolumeRayCastMapper(_vtk.vtkFixedPointVolumeRayCastMapper, _Base
 
     def __init__(self, theme=None):
         """Initialize this class."""
-        super().__init__(theme)
+        super().__init__(theme=theme)
 
 
 class GPUVolumeRayCastMapper(_vtk.vtkGPUVolumeRayCastMapper, _BaseVolumeMapper):
     """Wrap _vtk.vtkGPUVolumeRayCastMapper."""
 
     def __init__(self, theme=None):
         """Initialize this class."""
-        super().__init__(theme)
+        super().__init__(theme=theme)
 
 
 class OpenGLGPUVolumeRayCastMapper(_vtk.vtkOpenGLGPUVolumeRayCastMapper, _BaseVolumeMapper):
     """Wrap _vtk.vtkOpenGLGPUVolumeRayCastMapper."""
 
     def __init__(self, theme=None):
         """Initialize this class."""
-        super().__init__(theme)
+        super().__init__(theme=theme)
 
```

Comment:
Codecov says these 3 `super().__init__()` calls are uncovered, I assume this is because of OSMesa. Since this is pulling down coverage in this PR: should we no-cover these three classes entirely?

Reply:
Actually, do we even need these `__init__` definitions? Looks like they don't do anything compared to the parent super?

Reply:
> Actually, do we even need these __init__ definitions? Looks like they don't do anything compared to the parent super?

I'll check, but I think you're right, we don't need them.

Reply:
My local tests run fine without them, but I don't have a GPU, and neither will CI catch if anything is wrong here. Could you perhaps fire up the test suite? I'll push a commit to remove these inits and we'll see.